packages/backend/src/accounting/psql/ledger-transfer/model.ts

```diff
@@ -3,12 +3,7 @@ import { Model, QueryContext } from 'objection'
 import { Asset } from '../../../asset/model'
 import { BaseModel } from '../../../shared/baseModel'
 import { LedgerAccount } from '../ledger-account/model'
-
-export enum LedgerTransferState {
-  PENDING = 'PENDING',
-  POSTED = 'POSTED',
-  VOIDED = 'VOIDED'
-}
```

Comment:
Let's have the services import from the models, instead of the other way around. To keep the different layers in order. 

Or we re-delcare them in the services, like you've done for `LedgerTransferType` it looks like